examples/create-react-app-with-styled-components-typescript/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "extends": "./tsconfig.paths.json",
```

Comment:
Any particular reason to split out the paths to its own file?

Reply:
Yes, currently tsconfig paths are not supported:

```
The following changes are being made to your tsconfig.json file:
  - compilerOptions.paths must not be set (aliased imports are not supported)
```

This is kind of an override to this. See https://stackoverflow.com/questions/57070052/create-react-app-typescript-3-5-path-alias